client/view/Main.html

```diff
@@ -1,8 +1,32 @@
 <div class="top-bar">
-	<img src="/static/image/dragon-sword.svg" class="big-screen-logo" />
-	<img src="/static/image/dragon-sword-wide.svg" class="small-screen-logo" />
-	<div class="menu-container">
-		<Menu asr="{{asr}}" />
+	<a href="https://biblicalblueprints.com/" class="link-no-underline">
+		<img src="/static/image/bb-logo-2020-square.svg" class="big-screen-logo" />
+		<img src="/static/image/bb-logo-2020-square.svg" class="small-screen-logo" />
```

Comment:
Since these are both the same image, we should only need one – the only reason there were two images before was because the actual image used was different on differently-sized screens.

Reply:
Yes, it looks fine on smaller screens. Yes, the bb-logo.png isn't being unused. I used it to generate favicons. I can remove it.

Reply:
Fixed to one image, removed bb-logo.png.